### DISCUSSION TITLE
Android VPN Hotspot

### DISCUSSION THREAD

**Discussion by mushonnip** [xjasonlyu/tun2socks#153] [Q&A]
I am about to connect my android vpn connection to linux desktop,
there is a way to share the connection via socks5, for this case i use Every Proxy app,
my android socks5 server is 192.168.43.1:1080

I was successfull using badvpn-tun2socks before, but it always failed this time, so i looking for some alternative.
the socks5 server is working, i have tested it with Firefox proxy configuration. but my goal is to make it systemwide proxy.
can this project solve my problem? and how to configure it

thank you

**Comment by xjasonlyu** [comment]
Of course you can do it with this project, just configure it as usual. :)

**Reply by mushonnip** [reply]
https://github.com/xjasonlyu/tun2socks/wiki/Examples
I have tried this example, but not working,
I have no knowledge about tun interface, iptables or other network stuff, i usually using example or someone script.
Hopefully there is some example match with my case
